p2p/enode/urlv4.go

```diff
@@ -125,11 +127,14 @@ func parseComplete(rawurl string) (*Node, error) {
 		return nil, fmt.Errorf("invalid public key (%v)", err)
 	}
 	// Parse the IP address.
-	ips, err := net.LookupIP(u.Hostname())
-	if err != nil {
-		return nil, err
+	ip := net.ParseIP(u.Hostname())
+	if ip == nil {
+		ips, err := lookupIPFunc(u.Hostname())
+		if err != nil {
+			return nil, err
+		}
+		ip = ips[0]
```

Comment:
We should make sure that there is at least one ip in that list. In theory, not finding an IP address should return an error, but since this change is about switching resolvers, there is no guarantee that each resolver will respect this.

Reply:
Nvm, they'll have to make sure this doesn't happen and it should make a stack trace to point the way here.